index.bs

```diff
@@ -288,6 +288,7 @@ The Web Authentication API is defined by the union of the Web IDL fragments pres
 
 <pre class="idl">
     partial interface Navigator {
+        [SecureContext]
```

Comment:
> Should this be on the definition of interface WebAuthentication also (or instead)? 

yes, as also noted by YaronS over in issue #146.  this was a haste-makes-waste moment, sigh.   Ok, so we instead do this..
```
    [SecureContext]
    interface WebAuthentication {
        Promise < ScopedCredentialInfo > makeCredential (
            Account                                 accountInformation,
            sequence < ScopedCredentialParameters > cryptoParameters,
            BufferSource                            attestationChallenge,
            optional CredentialOptions              options
        );

        Promise < WebAuthnAssertion > getAssertion (
            BufferSource               assertionChallenge,
            optional AssertionOptions  options
        );
    };
```
And in #146 @mikewest notes..
> I'd also suggest using the [SecureContext] IDL attribute on entry points to the API like navigator.authentication, as well as visible interfaces like ScopedCredentialInfo, WebAuthentication, etc.

..so I'll add that also.